src/repository/repositories/users.repository.ts

```diff
@@ -13,4 +13,31 @@ export class UsersRepository {
 	async createUser(request: CreateUserRequest): Promise<User> {
 		return this.userModel.create({ ...request });
 	}
+
+	async findByEmail(email: string): Promise<User> {
```

Comment:
You can create method wirh parameters for example 
async findBy({email: string}: IFilter): Promise<User>